experimental/SymmetricIntersections/Types.jl

```diff
@@ -0,0 +1,198 @@
+import Oscar: elem_type
+
+"""
+We work always with finite groups and splitting fields of characteristic zero.
+So by default, we take `QQAb` which is algebraically close of characteristic
+zero.
+"""
+
+export CharacterGrassmannian,
+       DeterminantGrassmannian,
+       ElevCtx,
+       InvariantGrassmannian,
+       IsotypicalGrassmannian,
+       LinRep,
+       ParameterMap,
+       ProjRep,
+       RepRing,
+       SymmetricGrassmannian,
+       SymmetricIntersections
+
+###############################################################################
+#
+#  Elevations context
+#
+###############################################################################
+
+@attributes mutable struct ElevCtx{T, U}
+  L::Vector{T}
+  d::Int
+  it::SubObjectIterator{PointVector{fmpz}}
+  f::U
+  bounds::Tuple{Vector{Int}, Vector{Int}}
+
+  function ElevCtx(L::Vector{T}, d::Int, it::SubObjectIterator{PointVector{fmpz}}, f::U, bounds::Tuple{Vector{Int}, Vector{Int}}) where {T, U}
+    z = new{T, U}()
+    z.L = L
+    z.d = d
+    z.it = it
+    z.f = f
+    z.bounds = bounds
+    return z
+  end
+end
+
+###############################################################################
+#
+#  Representations
+#
+###############################################################################
+
+### Representation ring
+
+"""
+Here a representation ring is a "parent" for linear representation, used to store
+the underlying group, a splitting field (`QQAb` by default), a set of generators
+for the group, the character table, the irreducible characters and, iteratively,
+representations affording some of the irreducible characters (only the one needed
+so far). There could be less accessors since irreducible characters store the character
+tale which store the group and the field, by it is easily accessed like this.
```

Comment:
Typo? I am guessing...
```suggestion
tale which store the group and the field, but it is easily accessed like this.
```